File: UI/eturistickaagencija_mobile/lib/model/ocjena.dart
class Ocjena {
  final int? id;
  final int? korisnikId;
  final int? destinacijaId;
  final int? ocjenaUsluge;
  final String? komentar;

  Ocjena({
    this.id,
    this.korisnikId,
    this.destinacijaId,
    this.ocjenaUsluge,
    this.komentar,
  });

  factory Ocjena.fromJson(Map<String, dynamic> json) {
    return Ocjena(
      id: json['id'],
      korisnikId: json['korisnikId'],
      destinacijaId: json['destinacijaId'],
      ocjenaUsluge: json['ocjenaUsluge'],
      komentar: json['komentar'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'id': id,
      'korisnikId': korisnikId,
      'destinacijaId': destinacijaId,
      'ocjenaUsluge': ocjenaUsluge,
      'komentar': komentar,
    };
  }
}

File: eTuristickaAgencija.Service/Database/Ocjena.cs
using System;
using System.Collections.Generic;

namespace eTuristickaAgencija.Service.Database;

public partial class Ocjena
{
    public int Id { get; set; }

    public int OcjenaUsluge { get; set; }

    public string? Komentar { get; set; }

    public int? DestinacijaId { get; set; }

    public int? KorisnikId { get; set; }

    public virtual Destinacija? Destinacija { get; set; }

    public virtual Korisnik? Korisnik { get; set; }
}
